### PR TITLE
Translate api-reference.md

### PR DESCRIPTION
**Pull request by noisefilter19**

### CODE REVIEW

File: docs/docs/api-reference.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Gatsby API Reference
+title: Gatsby API संदर्भ
 overview: true
 ---
 
-Learn more about Gatsby API methods and options, common files, and customizations.
+Gatsby API विधियों और विकल्पों, सामान्य फ़ाइलों और अनुकूलन के बारे में अधिक जानें।
```
Comment (arshadkazmi42) [inline docs/docs/api-reference.md]: `विकल्पों` => `ऑप्शन्स`

Comment (arshadkazmi42) [inline docs/docs/api-reference.md]: `अनुकूलन` => `कस्टमाइज़ेशन`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
I have added two minor feedbacks. 
Can you work on fixing those.

**Comment by noisefilter19** [comment]
Done.

**Review by arshadkazmi42 — APPROVED** [review]
👍

**Comment by noisefilter19** [comment]
That was blazing fast! :)

**Comment by noisefilter19** [comment]
I'll take on something else

**Pull request merged**